development/storybook.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# capture yarn build output
+STORYBOOK_OUTPUT=$(yarn storybook:build 2>&1)
+# check for export warnings
+OUTPUT=$(echo "$STORYBOOK_OUTPUT" | grep 'WARN' | grep 'export')
```

Comment:
I'm guessing the output was this:

```
/bin/sh: 1: build-storybook: not found
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Maybe to catch this case, we should be looking for the exit code of the build command as well (`$?`)

Reply:
Marking this as resolved, as `set -e` should handle this.

Reply:
Where is the `set -e` that resolves this?

Reply:
It looks like it has been removed since I made that comment

Reply:
yes, I'm actually in the process of adding it back in now